doc-content/enterprise-only/openshift/server-immutable-s2i-deploy-assy.adoc

```diff
@@ -16,6 +16,8 @@ You can deploy an immutable {KIE_SERVER} using an S2I build. When you deploy the
 
 You cannot deploy or undeploy services on a running immutable {KIE_SERVER}. You can use {LOCAL_CENTRAL} to view monitoring information. The {KIE_SERVER} runs like any other pod on the OpenShift environment; you can use any container-based integration workflows as necessary.
 
+You can configure the immutable {KIE_SERVER} to receive and respond to API messages using JMS, communicating with an AMQ server.
```

Comment:
this sentence is not good, it is a little bit confusing.
Something like this sounds better:

"You can enable the JMS capabilities on the Kie Server Images to be able to interact with it through JMS API, requires external Messaging broker."